paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -169,6 +185,9 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
         supportFragmentManager.commit {
             when (transitionTarget) {
                 is PaymentOptionsViewModel.TransitionTarget.AddPaymentMethodFull -> {
+                    // Once the add fragment has been opened there is never a scenario that
```

Comment:
THis is needed for a bug where if you have a returning customer with saved cards and then you click add, and are on a LPM that is immediately complete (aka Affirm, or paypal), pressing back will re-launch the Add fragment.